### ISSUE TITLE
Add a way to display creatures weight & height in imperial units

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#84]
As a **User using the imperial units**
I would like **an option to display the creature weight & height in imperial units**
So that **I can manage them when it's needed**

## Things to know
PSDK has a code part that manage imperial units. `m` and `kg` are checked.
```ruby
         # Return the formated weight for Pokedex
          # @return [String]
          def pokedex_weight
            # @type [String]
            text = ext_text(9000, 70)
            using_imperial_unit = !text.downcase.end_with?('kg')
            format(text, using_imperial_unit ? (weight * 2.20462).ceil(2) : weight)
          end

          # Return the formated height for Pokedex
          # @return [String]
          def pokedex_height
            # @type [String]
            text = ext_text(9000, 71)
            using_imperial_unit = !text.downcase.end_with?('m')
            if using_imperial_unit
              inches = (height * 39.3701).to_i
              feet = inches / 12
              inches -= feet * 12
              format(text, feet, inches)
            else
              return format(text, height)
            end
          end
```

## Resources
[Link to the suggestion topic by Ixion83](https://discord.com/channels/143824995867557888/1097260174047187014) (Discord)

**Comment by mathys-mt** [comment]
Hi, 
I'd like to work on this issue if nobody is already fixing it.
Also, in which unit should i display the height ? Yard, foot or inch ?

**Comment by AerunDev** [comment]
> Hi, I'd like to work on this issue if nobody is already fixing it. Also, in which unit should i display the height ? Yard, foot or inch ?

Hello mathys,
I added a link to the Discord suggestion topic in the issue's description. This is a good question.
I also added some information about how PSDK manage the data on its side so maybe it answers the question.

Do you have an idea about how you'd like to implement it?
For this story, we'd like to implement a new app setting `Appearence` under a new submenu section called `APP SETTINGS` to add a way to choose between metric or imperial units. What do you think about it?

**Comment by mathys-mt** [comment]
Yeah, i was also thinking of creating a new setting page to implement this feature. From what i've seen, the only front-end files that would need to be updated are in PokemonDataBlock (to display height and weight in the new units and change it on the pokemon page). For this second change, i was just thinking of making a new little file to convert the units before their registration in the database, but keep registering it in the metric system in the database, and only displaying or editing it in both systems (maybe making another file would even not been needed since the imperial units are already implemented. 
I have not looked yet at the implementation of the languages settings page, but i think of implementing  system unit settings page the same way, and then use a bolean to check the selection and update the two pages mentioned above accordingly.
